src/plugins_3rdparty/primer3/src/Primer3Dialog.cpp

```diff
@@ -149,10 +187,40 @@ QString Primer3Dialog::intervalListToString(const QList<U2Region>& intervalList,
     return result;
 }
 
+QString Primer3Dialog::intListToString(const QList<int>& intList, const QString& delimiter) {
+    QString result;
+    bool first = true;
+    for (int num : intList) {
+        if (!first) {
+            result += " ";
+        }
+        result += QString::number(num);
+        result += delimiter;
+        first = false;
+    }
+    return result;
+}
+
+QString Primer3Dialog::okRegions2String(const QList<QList<int>>& regionLins) {
+    QString result;
+    bool first = true;
+    for (const auto& numList : regionLins) {
```

Comment:
As far as its a nit, I think, I lit it like this, because we almost have no simplifications in this case - we still need to iterate by `const QList<QList<int>>& regionLins` using `for`